src/System/useFeatureFlag.tsx

```diff
@@ -78,6 +79,7 @@ export function useTrackFeatureVariant({
   payload,
 }: VariantTrackingProperties) {
   const router = useRouter()
+  const experimentViewTracked = useRef<boolean>(false)
```

Comment:
This guards against sending an event on every react re-render. The experiment will fire once per component mount.